### ISSUE TITLE
Add XO config

### ISSUE THREAD

**Issue by BrainBuzzer** [fossasia/susi_desktop#109]
Travis and Codacy are two of the CI tools that we use. It seems like one of them wants developers to provide single quotes while other one wants double quotes. We want to fix this behavior by adding a config file and making sure everything is in flow.

##### Please check the applicable fields -

- [ ] I ensure that this issue is not a duplicate of any current open issues(s).
- [ ] Assign this issue to me (I want to work on this).
- [ ] I will send my PR only after I'm assigned to this issue by any maintainer.
- [ ] I understand that I will be unassigned from this issue if I fail to send a PR for this within a week.
